dl/src/main/scala/com/intel/analytics/bigdl/nn/TimeDistributed.scala

```diff
@@ -34,17 +35,23 @@ import scala.reflect.ClassTag
  * @tparam T
  */
 
-class TimeDistributed[T : ClassTag] ()
+class TimeDistributed[T : ClassTag] (layer: TensorModule[T])
 (implicit ev: TensorNumeric[T]) extends Container[Tensor[T], Tensor[T], T] {
```

Comment:
Otherwise, most of the methods should be overrided, such as parameters(), training(), evaluate(), getTimes(), etc. Might make it hard for other people to revise?